docker/Dockerfile

```
@@ -117,4 +117,3 @@ COPY --from=nemo-rl . /opt/nemo-rl
 # so do a quick check before trying to unshallow.
 RUN git rev-parse --is-shallow-repository | grep -q true && git fetch --unshallow || true
 RUN UV_LINK_MODE=symlink uv run nemo_rl/utils/prefetch_venvs.py
```

```suggestion
RUN UV_LINK_MODE=symlink uv run nemo_rl/utils/prefetch_venvs.py

```
nit: just so this file doesn't show up in the diff